### ISSUE TITLE
Incomplete documentation in Solving Problems with Time-dependent Hamiltonians page

### ISSUE THREAD

**Issue by iAditya09** [qutip/qutip-doc#128]
[http://qutip.org/docs/latest/guide/dynamics/dynamics-time.html](url)

Under section Accessing the state from solver, it says the state of the system can be obtained using args for the time dependent Hamiltonian, but I couldn't figure out what that key was to use the state of system in the time dependent Hamiltonian , an example would be good.

![qutipIssue](https://user-images.githubusercontent.com/7588933/109229114-d9eb3b00-7790-11eb-94ef-17d53aaabaf7.JPG)

**Comment by iAditya09** [comment]
@Ericgig

**Comment by Ericgig** [comment]
You're right, the table listing the keyword does not show up in the final documentation and they are not up-to-date...
You can find the keywords in QobjEvo's Docstring: `help(qutip.QobjEvo)`.

**Comment by iAditya09** [comment]
@Ericgig thanks! found the keywords. I am having some issue with the state being returned though and I am unsure how to work around them. I basically want to calculate expect(Oper, args["state"]) which gives me a dimension mismatch, if I try doing (Oper*args["state"]).tr() to find the expectation value, it tells me there is a shape mismatch. I initially start with Oper and psi0 shape as (8,8) but the args["state"] returns a shape of (64,1). Any idea how to get around this issue? states in the 1d vector or the 2d matrix form also doesn't help leading to similar dimension mismatch errors. I was trying to look into the source code but couldn't figure out how to unflatten the Qobj state

**Comment by Ericgig** [comment]
In the latest version, the `Qobj` should be reshaped to (8,8)...
Could you send me your `mesolve` call and qutip version.

In your case, my guess is that you have `oper` not a super operator but state as a vector dm.
You could change `oper` with `oper = qutip.spre(oper)` then `expect(Oper, args["state"]) ` should work.

Or you could reshape the state to `(8,8)` with `vector_to_operator(state)`. Then `(Oper*args["state"]).tr()` should work
